source/style-guide/information-types/index.txt

```diff
@@ -0,0 +1,70 @@
+.. _information-types:
+
+=================
+Information Types
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/info-typing/fact-general-guidance.rst
+
+The following sections describe information typing and provide
+prototypes that show how to apply this strategy to our documentation.
+
+What is Information Typing?
+---------------------------
+
+Information typing categorizes content by type based on your audience's
+goals. Throughout the writing process, information typing helps keep the
+information that you write focused on those goals.
+
+Additionally, information typing accomplishes the following goals:
+
+- Separate content into categories and delivers roughly one type at a
+  time.
+
+- Account for the purpose of a page before you write it.
+
+- Answer the user question "why do I care?" or "what does this mean for
+  me?" quickly.
+
+- Use a suggested format to help you get started on a blank page.
+
+What Information Types Do We Use?
+---------------------------------
+
+.. note:: DITA Document Types
+
+   The information types used in MongoDB documentation are similar to
+   `DITA Document Types
+   <http://docs.oasis-open.org/dita/dita/v1.3/errata02/os/complete/part3-all-inclusive/archSpec/technicalContent/dita-technicalContent-InformationTypes.html#dita_technicalContent_InformationTypes>`__.
+   While we do not impose the same {+rst-abbrev+} requirements that are
+   used in DITA, the high-level purposes of corresponding content types
+   are largely the same.
+
+Currently, some teams use the following information types:
+
+Concept
+  .. include:: /includes/info-typing/concept-overview.rst
+
+Reference
+  A reference page delivers granular details about a topic. The audience
+  includes experienced users. Reference pages may describe the use of a
+  method, operator, or database command.
+
+Task
+  A task page helps a user do a job. The audience includes users who
```

Comment:
Suggestion:


I would recommend replacing the first sentence with  "A task page instructs a user on how to accomplish a particular goal." I would also recommend changing the second sentence to identify the audience by level of experience.

Reasons for change:
This stuck out to me in a prior reading.
I think "job" could be interpreted as "occupation" which might not be intended. I think the next sentence includes similar information that could be consolidated rather than restated.

I think the audience identification should be consistent with the Reference type entry, which currently uses level of experience to identify them ("experienced users"). I think that better aligns with the [MongoDB personas](https://wiki.corp.mongodb.com/display/DESIGN/MongoDB+Personas) that our company uses for targeting.

Reply:
Agreed with your points here. Updated to clarify that tasks can be written for users of all experience levels.

Reply:
Also moving to an include to keep consistent with the task page.